modules.json

```diff
@@ -10,7 +10,7 @@
         "AllowPrerelease":false
     },
     "PSReadLine":{
-        "MinimumVersion":"2.0.0-beta6",
+        "MinimumVersion":"2.0.0-rc1",
         "MaximumVersion":"2.1",
```

Comment:
Just curious, why there are two `maximumversion` entries?

Reply:
@daxian-dbw where do you see a second maximum version entry?

Reply:
My bad, I misread the MinimumVersion entry :)